src/api/XRCompositionLayerPolyfill.ts

```diff
@@ -479,6 +482,8 @@ export default class XRCompositionLayerPolyfill implements XRCompositionLayer {
 				textureFormat === this.context.DEPTH24_STENCIL8 ||
 				textureFormat === this.context.DEPTH_STENCIL
 			) {
+				internalFormat = this.context.DEPTH24_STENCIL8
+                		textureFormat = this.context.DEPTH_STENCIL
```

Comment:
Can you explain why you force change the values of `internalFormat` and `textureFormat` here? I mostly want to know where in the spec this comes from; I'm much more familiar with WebGL1 than WebGL2 so some of the texture and internal format changes are new to me.

Reply:
Look to my draft PR, it doing similar:

https://github.com/immersive-web/webxr-layers-polyfill/pull/29/commits/d41c92d68a6e50b30977764b1f542feaa5bc6b9f

Reply:
I know; in general, there's a lot around WebGL2's internal and texture formats that need to be changed in this polyfill, and I'll have to think about how to do so. But I personally just don't know where in the spec the DEPTH changes are coming from; the RGB ones I know are from https://registry.khronos.org/webgl/specs/latest/2.0/#TEXTURE_TYPES_FORMATS_FROM_DOM_ELEMENTS_TABLE.

Are there any other depth format issues that I'd need to be aware of for WebGL2?

Reply:
When i tried to find a khronos docs, i find a table on webgl2fundamentals. It looks like valid.

in Opengl spec
https://registry.khronos.org/OpenGL-Refpages/es3.0/html/glRenderbufferStorage.xhtml